### DISCUSSION TITLE
RE4 motion controls? 🥺

### DISCUSSION THREAD

**Discussion by netriosilver** [praydog/REFramework#857] [General Questions]
🥺🥺🥺🥺🥺🥺

**Comment by netriosilver** [comment]
🥺🥺🥺🥺🥺🥺🥺🥺🥺🥺🥺🥺

**Comment by Draverd** [comment]
I have been asking the same thing, but no reply. It would be really appreciated praydog because the only other alternative would be to go buy a PS5, and PlaystationVR2. Which would be expensive just for that use. If you can praydog please take a break on what you are working on to make this a reality. 

Thank You
